src/components/DropDown/DropDown.jsx

```diff
@@ -77,16 +79,17 @@ export default class DropDown extends Component {
       children,
       direction,
       pullRight,
+      shade,
       title
     } = this.props;
     const {isOpen} = this.state;
 
     return (
-      <div className={`${styles.container} ${styles[direction]} ${pullRight ? styles.pullRight : ''}`}>
+      <div className={`${styles.container} ${styles[direction]} ${pullRight ? styles.pullRight : ''} ${shade === 'light' ? styles.lightShade : ''}`}>
```

Comment:
You know the shade will be either light or dark so I would just change this to `${styles[shade]}` and change the class from lightShade to light or dark in the scss file.